syntax = "proto3";

package org.apache.dubbo.samples.proto;

message GenerateRequest {
  string prompt = 1;
}

message GenerateReply {
  string message = 1;
}

service LlmService {

  rpc generate(GenerateRequest) returns (stream GenerateReply);
}
